src/content/en/updates/2016/07/capture-stream.md

```diff
@@ -0,0 +1,104 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The captureStream() method enables a MediaStream to be captured from a canvas, audio or video element, on Android and desktop.
+
+{# wf_updated_on: 2016-10-20 #}
+{# wf_published_on: 2016-10-20 #}
+{# wf_tags: canvas,chrome53,gif,media,security,webrtc #}
+{# wf_featured_image: /web/updates/images/2016/07/autoplay/featured.jpg #}
```

Comment:
I **strongly** recommend also including a `{# wf_featured_snippet: blah blah blah #} tag. It is used on the index page as the description. It can be longer than the meta description, and also include HTML. This is your hook to entice people to read your post.